templates/graph.html

```diff
@@ -0,0 +1,21 @@
+{% extends "page.html" %}
+{% block head %}
+  <title>Home page | F# Snippets</title>
+{% endblock %}
+
+{% block customPageScripts %}
+<script type="text/javascript" src="https://www.google.com/jsapi"></script>
```

Comment:
no it works fine, but now I have to add this boilerplate to all html files.
a property with this part would be useful for me.

2015-12-07 17:40 GMT+01:00 Taha Hachana <notifications@github.com>:

> In templates/graph.html
> <https://github.com/tpetricek/FsSnip.Website/pull/46#discussion_r46843249>
> :
>
> > @@ -0,0 +1,21 @@
> > +{% extends "page.html" %}
> > +{% block head %}
> > +  <title>Home page | F# Snippets</title>
> > +{% endblock %}
> > +
> > +{% block customPageScripts %}
> > +<script type="text/javascript" src="https://www.google.com/jsapi"></script>
>
> For server side usage and IDE integration I left loading the appropriate
> visualization packages to the user. According to the screenshots below this
> wasn't a showstopper for you.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tpetricek/FsSnip.Website/pull/46/files#r46843249>.
>